Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN set -ex \
 RUN mkdir /config
 COPY /config/requirements.txt /config/
 RUN pip install -r /config/requirements.txt
+COPY /config/firebase.json /config/firebase.json
```

Comment:
I would propose to mount or pass via ENV this config in the future.
In case we will publish our containers to the docker hub our firebase config would be open to everybody.